src/components/AccordionMenu/index.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { Button } from 'react-bootstrap';
+import { connect } from 'react-redux';
+import makeClassName from 'classnames';
+
+import { ApplicationState } from '../../reducers';
+import { ConnectedReduxProps } from '../../configureStore';
+import { actions, isExpanded } from '../../reducers/accordionMenu';
+import { AnyReactNode } from '../../typeUtils';
+import styles from './styles.module.scss';
+
+export const makeItemContentId = (title: string) => {
+  const id = title.replace(/[^A-Za-z0-9]+/g, '');
+  if (!id) {
+    throw new Error(`An ID could not be generated from this title: "${title}"`);
+  }
+  return `accordionContent${id}`;
+};
+
+export type PublicItemProps = {
+  children: AnyReactNode;
+  title: string;
+};
+
+export type DefaultItemProps = {
+  expanded: boolean;
```

Comment:
I guess that if one creates an `AccordionMenu` and specifies `expanded = true` on more than one `AccordionItem`, then the last one listed would "win"? Is this an issue, or just what one would expect?

Reply:
There isn't really an easy way to fix but I think it's OK. It doesn't seem like a hard thing to catch during development.